weblate/addons/models.py

```diff
@@ -66,21 +77,31 @@ class Meta:
         verbose_name_plural = "add-ons"
 
     def __str__(self):
-        return f"{self.addon.verbose}: {self.component}"
+        return f"{self.addon.verbose}: {self.project or self.component}"
```

Comment:
Nicer representation of site-wide add-ons:

```suggestion
        return f"{self.addon.verbose}: {self.project or self.component or 'site-wide'}"
```